backend/tests/services/test_local_device_command_service.py

```
@@ -163,6 +258,52 @@ def test_turn_file_changes_revert_applies_reverse_patch(tmp_path):
     assert (repo / "changed.txt").read_text(encoding="utf-8") == "before\n"
 
 
+def test_turn_file_changes_revert_applies_patch_sequence(tmp_path):
+    from app.services.device.command_registry import TURN_FILE_CHANGES_SCRIPT
+
+    repo, executor_home = _create_turn_file_changes_sequence_artifact(tmp_path)
+    result = subprocess.run(
```

Ruff (0.15.18)

[error] `subprocess` call: check for execution of untrusted input

(S603)